modepy/shapes.py

```diff
@@ -276,6 +277,34 @@ class Face:
     map_to_volume: Callable[[np.ndarray], np.ndarray]
 
 
+def face_normal(face: Face, normalize=True) -> np.ndarray:
+    """
+    .. versionadded :: 2021.2.1
+    """
+    volume_vertices = unit_vertices_for_shape(face.volume_shape)
+    face_vertices = volume_vertices[:, face.volume_vertex_indices]
+
+    if face.dim == 0:
+        # FIXME Grrrr. Hardcoded special case. Got a better idea?
+        (fv,), = face_vertices
+        return np.array([np.sign(fv)])
```

Comment:
It seems `dim == 0` likes to throw a wrench in quite a few places. However, I am fine with this personally!